cypress/integration/finance/funds/add-allocation-to-budget.spec.js

```diff
@@ -0,0 +1,81 @@
+import permissions from '../../../support/dictionary/permissions';
+import testType from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import Users from '../../../support/fragments/users/users';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import InteractorsTools from '../../../support/utils/interactorsTools';
+
+describe('ui-finance: Add budget to fund', () => {
+  const defaultfund = Funds.defaultUiFund;
```

Comment:
I suggest to copy this objects too like you copy { ...Ledgers.defaultUiLedger }

Reply:
Done